src/box/field_def.c

```diff
@@ -61,7 +61,8 @@ const uint32_t field_mp_type[] = {
 	/* [FIELD_TYPE_STRING]   =  */ 1U << MP_STR,
 	/* [FIELD_TYPE_NUMBER]   =  */ (1U << MP_UINT) | (1U << MP_INT) |
 		(1U << MP_FLOAT) | (1U << MP_DOUBLE),
-	/* [FIELD_TYPE_DOUBLE]   =  */ 1U << MP_DOUBLE,
+	/* [FIELD_TYPE_DOUBLE]   =  */ (1U << MP_UINT) | (1U << MP_INT) |
```

Comment:
I think there is something wrong with this solution:
```
tarantool> box.execute([[create table t(d double primary key);]])
---
- row_count: 1
...

tarantool> box.space.T:insert({1})
---
- [1]
...

tarantool> box.execute([[select d / 2 from t;]])
---
- metadata:
  - name: COLUMN_1
    type: double
  rows:
  - [0]
...
```

At the same time, in SQL:
```
tarantool> box.execute([[create table t(d double primary key);]])
---
- row_count: 1
...

tarantool> box.execute([[insert into t values (1);]])
---
- row_count: 1
...

tarantool> box.execute([[select d / 2 from t;]])
---
- metadata:
  - name: COLUMN_1
    type: double
  rows:
  - [0.5]
...
```